### PR TITLE
DOCS-1951-CE-3.18.1-GA

### PR DESCRIPTION
**Pull request by bmckercher123**
Product Version(s):
CE 3.18.1 (GA)

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

File: calico-enterprise_versioned_docs/version-3.18-2/variables.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
 const variables = {
-  releaseTitle: 'v3.18.0-2.0',
+  releaseTitle: 'v3.18.1',
   prodname: 'Calico Enterprise',
   prodnamedash: 'calico-enterprise',
   version: 'v3.18',
-  baseUrl: '/calico-enterprise/3.18',
-  filesUrl: 'https://downloads.tigera.io/ee/v3.18.0-2.0',
+  baseUrl: '/calico-enterprise/3.18-2',
```
Comment (ctauchen) [inline calico-enterprise_versioned_docs/version-3.18-2/variables.js]:
```suggestion
  baseUrl: '/calico-enterprise/3.18',
```

File: docusaurus.config.js [hunk 2]
```diff
@@ -373,27 +373,22 @@ const config = {
         path: 'calico-enterprise',
         routeBasePath: 'calico-enterprise',
         editCurrentVersion: true,
-        onlyIncludeVersions: ['3.18-2','3.17','3.16','3.15'],
+        onlyIncludeVersions: ['3.18','3.17','3.16','3.15'],
```
Comment (ctauchen) [inline docusaurus.config.js]:
```suggestion
        onlyIncludeVersions: ['3.18-2','3.17','3.16','3.15'],
```

File: docusaurus.config.js [hunk 3]
```diff
@@ -373,27 +373,22 @@ const config = {
...
         lastVersion: '3.17',
```
Comment (ctauchen) [inline docusaurus.config.js]:
```suggestion
        lastVersion: '3.18',
```

File: docusaurus.config.js [hunk 4]
```diff
@@ -373,27 +373,22 @@ const config = {
...
-            banner: 'unreleased',
-          },
-          3.18: {
-            label: '3.18 (early preview)',
-            path: '3.18',
-            banner: 'unreleased',
+          '3.18': {
+            label: '3.18 (latest)',
+            path: 'latest',
+            banner: 'none',
```
Comment (ctauchen) [inline docusaurus.config.js]:
```suggestion
          '3.18-2': {
            label: '3.18 (latest)',
            path: 'latest',
            banner: 'none',
```

File: docusaurus.config.js [hunk 5]
```diff
@@ -373,27 +373,22 @@ const config = {
...
           },
-          '3.18-2': {
-            label: '3.18 (early preview)',
-            path: '3.18',
-            banner: 'unreleased',
-          },
-          3.18: {
-            label: '3.18 (early preview)',
-            path: '3.18',
-            banner: 'unreleased',
```
Comment (ctauchen) [inline docusaurus.config.js]: stet

File: calico-enterprise_versioned_docs/version-3.18-2/variables.js (hunk 1 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.18-2/variables.js]: Fixed.

File: docusaurus.config.js (hunk 2 above)
Comment (bmckercher123) [reply docusaurus.config.js]: Fixed.

File: docusaurus.config.js (hunk 3 above)
Comment (bmckercher123) [reply docusaurus.config.js]: Fixed.

File: docusaurus.config.js (hunk 4 above)
Comment (bmckercher123) [reply docusaurus.config.js]: Fixed.

Comment (bmckercher123) [reply docusaurus.config.js]: Fixed.

File: calico-enterprise_versioned_docs/version-3.18-2/variables.js (hunk 1 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.18-2/variables.js]: @ctauchen baseUrl: '/calico-enterprise/3.18' did not work. Had to change it to 3.18.1 for manifests to show up correctly. Also, manifest URLs are all fine except helm...it is showing 3.18.2. I'll check with Dan to see if something is not changed yet on Eng side. Any ideas?

File: calico-enterprise_versioned_docs/version-3.18-2/variables.js [hunk 6]
```diff
@@ -15,7 +15,7 @@ const variables = {
   noderunning: 'calico-node',
   rootDirWindows: 'C:\\TigeraCalico',
   registry: 'quay.io/',
-  chart_version_name: 'v3.18.0-2.0-0',
+  chart_version_name: 'v3.18.1',
```
Comment (ctauchen) [inline calico-enterprise_versioned_docs/version-3.18-2/variables.js]: This doesn't seem right. I'd expect something like `v3.18.1-0`. Check with rene?

File: calico-enterprise_versioned_docs/version-3.18-2/variables.js [hunk 7]
```diff
@@ -1,12 +1,12 @@
 const releases = require('./releases.json');
 
 const variables = {
-  releaseTitle: 'v3.18.0-2.0',
+  releaseTitle: 'v3.18.1',
   prodname: 'Calico Enterprise',
   prodnamedash: 'calico-enterprise',
   version: 'v3.18',
   baseUrl: '/calico-enterprise/3.18',
```
Comment (ctauchen) [inline calico-enterprise_versioned_docs/version-3.18-2/variables.js]:
```suggestion
  baseUrl: '/calico-enterprise/latest',
```
ALSO: Change the equivalent 3.17 variables.js from latest -> 3.17

Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.18-2/variables.js]: Fixed.

**Comment by ctauchen** [comment]
LGTM

**Pull request merged**